langlib/langlib-test/src/test/resources/test-src/valuelib_test.bal

```diff
@@ -895,9 +921,10 @@ function testFromJsonStringWithTypeRecord() {
     string str = "{\"name\":\"Name\",\"age\":35}";
     Student3|error studentOrError = str.fromJsonStringWithType(Student3);
 
-    assert(studentOrError is Student3, true);
-    Student3 student = <Student3> studentOrError;
-    assert(student.name, "Name");
+    if (studentOrError is Student3) {
+        Student3 student = <Student3> studentOrError;
+        assert(student.name, "Name");
+    }
```

Comment:
We are losing the `studentOrError is Student3` assertion here, right? Why don't we just do

```ballerina
    assert(studentOrError is Student3, true);
    Student3 student = checkpanic studentOrError;
    assert(student.name, "Name");
```

Reply:
Fixed